torchbenchmark/e2e_models/hf_bert/__init__.py

```diff
@@ -1,5 +1,6 @@
 from accelerate.utils.dataclasses import DeepSpeedPlugin
 import torch
+import torch._dynamo
```

Comment:
Let's wrap this in a `try` block, since other users might test this code on stable/old pytorch releases, where `torch._dynamo` is not available.